lib/std/os.zig

```diff
@@ -5465,3 +5465,30 @@ pub fn prctl(option: i32, args: anytype) PrctlError!u31 {
         else => |err| return std.os.unexpectedErrno(err),
     }
 }
+
+pub const GetrlimitError = UnexpectedError;
+
+pub fn getrlimit(resource: rlimit_resource, limits: *rlimit) GetrlimitError!void {
+    const rc = system.getrlimit(resource, limits);
+    switch (errno(rc)) {
+        0 => return,
+        EFAULT => unreachable, // bogus pointer
+        EINVAL => unreachable,
+        else => |err| return std.os.unexpectedErrno(err),
+    }
+}
+
+pub const SetrlimitError = error{
+    PermissionDenied,
+} || UnexpectedError;
+
+pub fn setrlimit(resource: rlimit_resource, limits: *const rlimit) SetrlimitError!void {
```

Comment:
No need for a pointer parameter here:
```suggestion
pub fn setrlimit(resource: rlimit_resource, limits: rlimit) SetrlimitError!void {
```